ConsoleView.cpp

```diff
@@ -1,34 +1,66 @@
+#include "ConsoleView.h"
+#include "UMLRelationship.h"
+#include <iostream>
 //All these functions return void, they are simply here 
 //for the user to call when they want to get information on 
 //a CLASS (or a list of all classes), RELATIONSHIP, or ATTRIBUTE
 
+std::ostream& operator<<(std::ostream& out, const UMLRelationship& dt) {
+    
+    out << dt;
+    
+    return out;
+}
+
+std::ostream& operator<<(std::ostream& out, const std::vector<UMLAttribute>& vec) {
+    
+    out << vec;
+    
+    return out;
+}
+std::ostream& operator<<(std::ostream& out, const UMLClass& x) {
+    
+    out << x;
+    
+    return out;
+}
+
+std::ostream& operator<<(std::ostream& out, const UMLAttribute& dt)
+{
+    out << dt;
+    return out;
+}
+
+
+
+
 //Prtint class will be used to print the name of a class given a UMLClass object
 //By using the get_class_name function from the UMLClass, it then prints the name of the Class given
 //@Para: UMLClass OBJ
-void ConsoleView::PrintClassName(UMLClass X){
+void ConsoleView::Print_Class_Name(UMLClass X){
     
-    std::cout<< x.get_class_name(); 
+    std::cout<< X.get_class_name(); 
```

Comment:
I still dont love the X as the variable name in this function. Its extremely non-descriptive

Reply:
changed it to something more visible